packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.module.scss

```diff
@@ -6,6 +6,7 @@
     display: flex;
     margin: 0;
     max-width: 100%;  // Needed for IE10 truncation
+    min-width: calc(10em - 2px); // Must use ems for font scaling
```

Comment:
2px seems a little random. Can you explain this number calculation a little more?

Reply:
This is a minor extra protection assuming that someone might put this inside a cell and they set the overall width of the cell itself (borders included) to the min-width listed in the doc (10em or 140px) not account for the borders -- most cells will have a 1px border on the left and right. 

So the actual content width of the component has an absolute minimum size of 138px (@14px=1rem), even though consumers ideally would provide the actual content space of 140px.